ktor-server/ktor-server-core/jvm/src/io/ktor/server/response/ApplicationResponseFunctionsJvm.kt

```diff
@@ -52,6 +53,18 @@ public suspend fun ApplicationCall.respondFile(
     respond(message)
 }
 
+/**
+ * Responds to a client with a contents of a file with the name [fileName] in the [baseDir] folder
+ */
+public suspend fun ApplicationCall.respondFile(
+    baseDir: Path,
+    fileName: Path,
+    configure: OutgoingContent.() -> Unit = {}
+) {
+    val message = LocalFileContent(baseDir, fileName).apply(configure)
```

Comment:
I've noticed that `LocalFileContent` is deprecated in favor and `LocalPathContent` should be used instead. Should we also rename `respondFile` into `respondPath`?